sections/video.liquid

```diff
@@ -0,0 +1,141 @@
+            {%- if section.settings.cover_image.width >= 1440 -%}{{ section.settings.cover_image | img_url: '1440x' }} 1440w,{%- endif -%}
+            {%- if section.settings.cover_image.width >= 990 -%}{{ section.settings.cover_image | img_url: '990x' }} 990w,{%- endif -%}
+            {%- if section.settings.cover_image.width >= 2080 -%}{{ section.settings.cover_image | img_url: '2080x' }} 2080w{%- endif -%}"
+            src="{{ section.settings.cover_image | img_url: '533x' }}"
+            sizes="(min-width: 990px)990px, (min-width: 750px)calc(100vw - 100px), calc(100vw - 30px)"
+            alt="{{ section.settings.description | escape }}"
+            loading="lazy"
+            width="{{ section.settings.cover_image.width }}"
+            height="{{ section.settings.cover_image.height }}"
+            class="video-section__image"
+          >
+        {%- else -%}
+          {{ 'collection-2' | placeholder_svg_tag: 'placeholder-svg placeholder video-section__image' }}
+        {%- endif -%}
+        <span class="deferred-media__poster-button motion-reduce">
+          {%- render 'icon-play' -%}
+        </span>
+      </button>
+      <template>
+        {%- if section.settings.video_url.type == 'youtube' -%}
+          <iframe src="https://www.youtube.com/embed/{{ section.settings.video_url.id }}?enablejsapi=1" class="js-youtube" allow="autoplay; encrypted-media" allowfullscreen title="{{ section.settings.description | escape }}"></iframe>
+        {%- else -%}
+          <iframe src="https://player.vimeo.com/video/{{ section.settings.video_url.id }}" class="js-vimeo" allow="autoplay; encrypted-media" allowfullscreen title="{{ section.settings.description | escape }}"></iframe>
+        {%- endif -%}
+      </template>
+    </deferred-media>
+  </div>
+</div>
+
+{% schema %}
+{
+  "name": "Video",
+  "tag": "section",
+  "class": "spaced-section spaced-section--full-width",
+  "settings": [
+    {
+      "type": "text",
+      "id": "heading",
+      "default": "Video",
+      "label": "Heading"
+    },
+    {
+      "type": "image_picker",
+      "id": "cover_image",
+      "label": "Cover image"
+    },
+    {
+      "type": "video_url",
+      "id": "video_url",
+      "accept": [
+        "youtube",
+        "vimeo"
+      ],
+      "default": "https:\/\/www.youtube.com\/watch?v=_9VUPq3SxOc",
+      "label": "Link",
+      "placeholder": "Use a YouTube or Vimeo link",
+      "info": "Video will play inside the page"
+    },
+    {
+      "type": "text",
+      "id": "description",
+      "default": "Describe the video for customers using screen readers",
+      "label": "Video alt text"
+    },
+    {
+      "type": "checkbox",
+      "id": "image_padding",
+      "label": "Add image padding",
+      "default": false,
+      "info": "Select image padding if you don't want your images to be cropped"
```

Comment:
```suggestion
      "info": "Select image padding if you don't want your cover image to be cropped."
```
Aligned with @gretchen-willenberg for changing this and update on the Collage section as well. This setting is applied to one image at once and it appears to be the cover image. Reading is clearer for the merchant.

Reply:
Sounds good. I will need to add the height limit. Right now it's adapting to whatever the size of the image is.